src/ua/lviv/iot/algo/part1/lab1/Plate.java

```diff
@@ -0,0 +1,47 @@
+package ua.lviv.iot.algo.part1.lab1;
+import lombok.*;
```

Comment:
https://www.google.com/url?sa=t&source=web&rct=j&url=https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad&ved=2ahUKEwjj6vrrxc_9AhWRmosKHd_lBBgQFnoECEEQAQ&usg=AOvVaw1xfzwHbr4yGZZur_mRISEP